src/NexusMods.Paths/FileSystemAbstraction/FileVersionInfo.cs

```diff
@@ -0,0 +1,9 @@
+namespace NexusMods.Paths;
+
+public record FileVersionInfo(Version ProductVersion)
```

Comment:
Please use `record struct` here, as this element composes of only 1 item [which itself is heap allocated] and does not need to be heap allocated.

Don't forget the docs 😉